manifest.json

```diff
@@ -49,7 +49,8 @@
         "*://store.steampowered.com/join/*",
         "*://store.steampowered.com/api/*",
         "*://support.steampowered.com/*",
-        "*://help.steampowered.com/*"
+        "*://help.steampowered.com/*",
+        "*://translation.steampowered.com/*"
```

Comment:
```suggestion
        "*://translation.steampowered.com/*",
        "*://partner.steampowered.com/*"        
```

Reply:
I thought the Steamworks site is https://partner.steamgames.com/ ? Anyway, added, thanks!